runtime/doc/vietnamese.txt

```diff
@@ -0,0 +1,87 @@
+*vietnamese.txt*   Vim
+
+
+		  VIM REFERENCE MANUAL    by Phạm Bình An
+
+
+Vietnamese language localization and support in Vim	  *vietnamese* *Vietnamese*
+
+                                      Type |gO| to see the table of contents.
+
+===============================================================================
+1. Introduction
+							*vietnamese-intro*
+
+Vim supports Vietnamese language in the following ways:
+
+- Built-in |Vietnamese-keymap|, which allows you to type Vietnamese characters
+  in |Insert-mode| and |search-commands| using US keyboard layout.
+- Localization in Vietnamese. See |vietnamese-l18n|
+
+===============================================================================
+2. Vietnamese keymaps
+							*vietnamese-keymap*
+
+To switch between languages you can use your system native keyboard switcher,
+or use one of the Vietnamese keymaps included in the Vim distribution, like
+below >vim
```

Comment:
```suggestion
below >
```

Reply:
I use `>vim` there because if one day Vim integrates treesitter, then that codeblock will be highlighted like a normal Vimscript file, which makes help doc easier to read. 

Of course all Vimscript code blocks in this file are single-line, I guess it is already easy enough to read

Reply:
That does not look likely at the moment, and Vim's syntax file is not able to conceal these.